src/Service/RunSqlCommandFactory.php

```diff
@@ -25,6 +25,7 @@ public function __invoke(ContainerInterface $serviceLocator, $requestedName, ?ar
             );
         }
 
+        /** @psalm-suppress TooFewArguments */
```

Comment:
Instead of Psalm annotations this should be put into the psalm config.

Reply:
Do you mean putting it in the baseline?

Reply:
Yes, looks like the baseline file is supposed to contain it.

Reply:
Actually, I removed the whole check for the presence of SingleConnectionProvider, as that was introduced in dbal 2.11.0 (see [dbal@b10d4db8](https://github.com/doctrine/dbal/commit/b10d4db882eeaeda044aa71946d87882a244a094)) and we do not support earlier versions anymore.